config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_gkehubfeaturememberships.gkehub.cnrm.cloud.google.com.yaml

```diff
@@ -527,18 +530,11 @@ spec:
                   If this is equal to metadata.generation, then that means that the
                   current reported status reflects the most recent desired state of
                   the resource.
+                format: int64
                 type: integer
             type: object
-        required:
```

Comment:
We probably want to mark spec as required.  Because we're using the latest and greatest controller-gen, you should be able to use the simpler:

```
// +required
```

(instead of the more complex kubebuilder annotation)